libfwupdplugin/fu-rustgen-struct.h.in

```diff
@@ -1,5 +1,7 @@
 typedef GByteArray {{obj.name}};
-G_DEFINE_AUTOPTR_CLEANUP_FUNC({{obj.name}}, g_byte_array_unref)
+
+#define {{obj.c_method('Unref')}} g_byte_array_unref
```

Comment:
Do you plan to drag `unref` back to 19x? That'll be very helpful to fix this issue out there.

Reply:
sure thing, can do.

Reply:
https://github.com/fwupd/fwupd/pull/8167